cg/services/run_devices/abstract_classes.py

```diff
@@ -1,11 +1,14 @@
 """Post-processing service abstract classes."""
 
+import logging
 from abc import ABC, abstractmethod
 from pathlib import Path
 
 from cg.services.run_devices.abstract_models import PostProcessingDTOs, RunData, RunMetrics
 from cg.services.run_devices.constants import POST_PROCESSING_COMPLETED
 
+LOG = logging.getLogger(__name__)
```

Comment:
Seems like we are not using the LOG in this file anymore

Reply:
fixed